src/recipe.rs

```diff
@@ -69,8 +69,15 @@ impl<'src, D> Recipe<'src, D> {
   }
 
   pub(crate) fn confirm(&self) -> RunResult<'src, bool> {
-    if self.attributes.contains(&Attribute::Confirm) {
-      eprint!("Run recipe `{}`? ", self.name);
+    if let Some(Attribute::Confirm(prompt_text)) = self
```

Comment:
Let's do something like:

```
for attribute in self.attributes {
  if let Attribute::Confirm(prompt) = attribute {
    // print prompt
    break;
  }
}
``

Reply:
I simplified the logic with a loop like you suggested